src/main/java/seedu/dictionote/logic/parser/DictionoteParser.java

```diff
@@ -49,6 +50,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddCommand.COMMAND_WORD:
             return new AddCommandParser().parse(arguments);
 
+        case AddNoteCommand.COMMAND_WORD:
+            return new AddNoteCommandParser().parse(arguments);
```

Comment:
Update the test case on DictionoteParserTest? I notice there no test added, please add all the required test for all the method you added, if applicable.